.github/workflows/ci.yml

```diff
@@ -36,13 +36,13 @@ jobs:
             run: 'xbstrap init ../src'
             working-directory: build/
           - name: Build mlibc
-            run: 'xbstrap install ${{matrix.builds}}'
+            run: 'C=gcc-10 CXX=g++-10 xbstrap install ${{matrix.builds}}'
             working-directory: build/
           - name: Test mlibc
-            run: 'meson test -C pkg-builds/${{matrix.builds}}'
+            run: 'C=gcc-10 CXX=g++-10 meson test -C pkg-builds/${{matrix.builds}}'
             working-directory: build/
           - name: Build GCC
-            run: 'xbstrap install-tool gcc'
+            run: 'C=gcc-10 CXX=g++-10 xbstrap install-tool gcc'
```

Comment:
We should pass the compiler environment vars in `bootstrap.yml` (to the `meson` invocation when compiling mlibc).